numba_dppy/rename_numpy_functions_pass.py

```diff
@@ -0,0 +1,117 @@
+from numba.core import ir
+from numba.core.compiler_machinery import FunctionPass, register_pass
+from numba.core.ir_utils import find_topo_order, mk_unique_var, simplify_CFG
+import numba_dppy
+
+rewrite_function_name_map = {"sum": (["np"], "sum"), "eig": (["linalg"], "eig")}
+
+
+class RewriteNumPyOverloadedFunctions(object):
+    def __init__(self, state, rewrite_function_name_map=rewrite_function_name_map):
+        self.state = state
+        self.function_name_map = rewrite_function_name_map
+
+    def run(self):
+        """
+        This function rewrites the name of NumPy functions that exist in self.function_name_map
```

Comment:
Thanks for the nice docstring. We can run black for the new file. The comments do not conform tho PEP-8

Reply:
Ran black 20.8b1 still did not reformat it. I manually made the line length smaller.